.github/workflows/ci.yml

```diff
@@ -0,0 +1,205 @@
+name: CI
+on:
+  push:
+  pull_request:
```

Comment:
This is just temporary i guess? noticed that lots of duplicated actions were triggered

Reply:
Yes, it's temporary. Addressed in https://github.com/jqlang/jq/pull/2620/commits/69e27dde737bd258522d3b07528efe63d7aaef9d